src/test/librados/io.cc

```diff
@@ -826,6 +827,12 @@ TEST_F(LibRadosIoEC, OverlappingWriteRoundTrip) {
   char *buf = (char *)new char[dbsize];
   char *buf2 = (char *)new char[bsize];
   char *buf3 = (char *)new char[dbsize];
+  auto lam = [&] {
```

Comment:
Well, yes, we could change everything to use the stack, that would be easy, but I figured I would follow the lead set by the original author.